scripts/config/keyboardArray.js

```diff
@@ -0,0 +1,360 @@
+const keyboardArray = [
+  {
+    code: 'Backquote',
+    ru: 'ё',
+    en: '`',
+    alt: '~',
+  },
+  {
+    code: 'Digit1',
+    ru: '1',
+    en: '1',
+    alt: '!',
+  },
+  {
+    code: 'Digit2',
+    ru: '2',
+    en: '2',
+    alt: '@',
+  },
+  {
+    code: 'Digit3',
+    ru: '3',
+    en: '3',
+    alt: '#',
+  },
+  {
+    code: 'Digit4',
+    ru: '4',
+    en: '4',
+    alt: '$',
+  },
+  {
+    code: 'Digit5',
+    ru: '5',
+    en: '5',
+    alt: '%',
+  },
+  {
+    code: 'Digit6',
+    ru: '6',
+    en: '6',
+    alt: '^',
+  },
+  {
+    code: 'Digit7',
+    ru: '7',
+    en: '7',
+    alt: '&',
+  },
+  {
+    code: 'Digit8',
+    ru: '8',
+    en: '8',
+    alt: '*',
+  },
+  {
+    code: 'Digit9',
+    ru: '9',
+    en: '9',
+    alt: '(',
+  },
+  {
+    code: 'Digit0',
+    ru: '0',
+    en: '0',
+    alt: ')',
+  },
+  {
+    code: 'Minus',
+    ru: '-',
+    en: '-',
+    alt: '_',
+  },
+  {
+    code: 'Equal',
+    ru: '=',
+    en: '=',
+    alt: '+',
+  },
+  {
+    code: 'Backspace',
+    label: 'Backspace',
+    isOperating: true,
+    additionalClasses: ['keyboard-button_name-backspace', 'keyboard-button_dark'],
+  },
+  {
+    code: 'Tab',
+    label: 'Tab',
+    isOperating: true,
+    content: '\t',
+    additionalClasses: ['keyboard-button_dark'],
+  },
+  {
+    code: 'KeyQ',
+    ru: 'й',
+    en: 'q',
+  },
+  {
+    code: 'KeyW',
+    ru: 'ц',
+    en: 'w',
+  },
+  {
+    code: 'KeyE',
+    ru: 'у',
+    en: 'e',
+  },
+  {
+    code: 'KeyR',
+    ru: 'к',
+    en: 'r',
+  },
+  {
+    code: 'KeyT',
+    ru: 'е',
+    en: 't',
+  },
+  {
+    code: 'KeyY',
+    ru: 'н',
+    en: 'y',
+  },
+  {
+    code: 'KeyU',
+    ru: 'г',
+    en: 'u',
+  },
+  {
+    code: 'KeyI',
+    ru: 'ш',
+    en: 'i',
+  },
+  {
+    code: 'KeyO',
+    ru: 'щ',
+    en: 'o',
+  },
+  {
+    code: 'KeyP',
+    ru: 'з',
+    en: 'p',
+  },
+  {
+    code: 'BracketLeft',
+    ru: 'х',
+    en: '[',
+  },
+  {
+    code: 'BracketRight',
+    ru: 'ъ',
+    en: ']',
+  },
+  {
+    code: 'Backslash',
+    ru: '\\',
+  },
+  {
+    code: 'Delete',
+    label: 'Del',
+    isOperating: true,
+    additionalClasses: ['keyboard-button_dark'],
+  },
+  {
+    code: 'CapsLock',
```

Comment:
A bug:

* CapsLock behaves incorrectly.

<img width="762" alt="s_caps" src="https://user-images.githubusercontent.com/3214353/79079161-1f884c00-7d16-11ea-8c52-18abf258db8b.png">

Reply:
Is is strange. It works for me

Reply:
Expected result: clicking on caps enabled/disabled UPPERCASE
Actual result: to switch a keyboard to UPPERCASE/lowercase it's needed to press on caps twice.

Reply:
Done

Reply:
Issue with CAPS is still reproduced, but this is Mac-elated issue, so need to fix.